src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -6,10 +6,13 @@
  */
 public class Bulldozer extends Machine {
     public Bulldozer() {
```

Comment:
Don't remove the no-args constructor, but also make sure it doesn't contain unnecessary logic as currently seen in line 9.